src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/AncientPedestalTask.java

```diff
@@ -0,0 +1,36 @@
+package io.github.thebusybiscuit.slimefun4.implementation.tasks;
+
+import java.util.Map;
+import java.util.Optional;
+
+import org.bukkit.Location;
+import org.bukkit.entity.Item;
+
+import io.github.bakedlibs.dough.blocks.BlockPosition;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunItems;
+import io.github.thebusybiscuit.slimefun4.implementation.items.altar.AncientPedestal;
+
+public class AncientPedestalTask implements Runnable {
+
+    private final AncientPedestal pedestalItem = (AncientPedestal) SlimefunItems.ANCIENT_PEDESTAL.getItem();
```

Comment:
This should probably be a param for the constructor instead of hardcoding this here.